storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/integration/GetOperationTest.java

```diff
@@ -165,4 +165,34 @@ public void getListOfObjectsAsObservableOnlyInitialValue() {
         changesTestSubscriber.assertNoErrors();
         changesTestSubscriber.assertValues(Changes.newInstance(TestItem.CONTENT_URI));
     }
+
+    @Test
+    public void getNumberOfResults() {
+        TestSubscriber<Changes> changesTestSubscriber = new TestSubscriber<Changes>();
+
+        storIOContentResolver
+                .observeChangesOfUri(TestItem.CONTENT_URI)
+                .take(1)
```

Comment:
Sure, no problem, but don't forget to override it in ContentProvider because by default it'll just call `insert` in a loop and it'll create 8 notifications :)